src/topp/MapAlignerIdentification.cpp

```diff
@@ -158,11 +158,27 @@ class TOPPMapAlignerIdentification :
                          vector<TransformationDescription>& transformations,
                          Int reference_index)
   {
-    algorithm.align(data, transformations, reference_index);
-
     // find model parameters:
     Param model_params = getParam_().copy("model:", true);
     String model_type = model_params.getValue("type");
+
+    try
+    {
+      algorithm.align(data, transformations, reference_index);
+    }
+    catch (Exception::MissingInformation& err)
+    {
+      if (getFlag_("force"))
+      {
+        OPENMS_LOG_ERROR
+          << "Error: alignment failed. Details:\n" << err.getMessage()
+          << "\nSince 'force' is set, processing will continue using 'identity' transformations."
+          << endl;
+        model_type = "identity";
+      }
+      else throw(err);
```

Comment:
better rethrow with throw()
https://www.learncpp.com/cpp-tutorial/14-6-rethrowing-exceptions/

Reply:
Thanks for the hint! (I tried `throw()` initially, but that didn't compile...)